### ISSUE TITLE
Confusing log messages about logging

### ISSUE THREAD

**Issue by rfc1036** [NICMx/FORT-validator#103]
I have switched the Debian package systemd unit to use:

```
ExecStart=/usr/bin/fort --log.output=syslog --validation-log.output=syslog --configuration-file /etc/fort/config.json
```

in the hope of just sending everything to syslog and cleaning up the startup log messages, but they are still confusing:

```
Dec 04 13:49:46 bongo.bofh.it fort[543526]: INF: Disabling validation logging on syslog.
Dec 04 13:49:46 bongo.bofh.it fort[543526]: Dec  4 13:49:46 INF: Disabling validation logging on syslog.
Dec 04 13:49:46 bongo.bofh.it fort[543526]: Dec  4 13:49:46 INF: Disabling validation logging on standard streams.
Dec 04 13:49:46 bongo.bofh.it fort[543526]: Dec  4 13:49:46 INF: Syslog log output configured; disabling operation logging on standard streams.
Dec 04 13:49:46 bongo.bofh.it fort[543526]: Dec  4 13:49:46 INF: (Operation Logs will be sent to syslog only.) 
Dec 04 13:49:46 bongo.bofh.it fort[543526]: INF: Disabling validation logging on standard streams.
Dec 04 13:49:46 bongo.bofh.it fort[543526]: INF: Syslog log output configured; disabling operation logging on standard streams.
Dec 04 13:49:46 bongo.bofh.it fort[543526]: INF: (Operation Logs will be sent to syslog only.)
```

I believe that if syslog output is enabled on the command line then fort should be able to not report these useless messages about switching the output back and forward.

Also, there is no point in reporting in syslog that messages are being sent to syslog and on the console that messages are being sent to the console because this is self-evident.

(I am not using `--daemon` because having systemd monitor a non-forking process is simpler and more reliable.)

Also, I do not believe that errors to syslog should have these `INF` and `ERR` prefixes, because syslog already has out of band metadata which represent messages severity:

```
Dec 04 13:51:01 bongo.bofh.it fort[543526]: ERR: /etc/tals/apnic.tal: Cannot generate rsync://rpki.co/repo/'s cage. I'm probably going to end up deleting it from the cache.
```

**Comment by ydahhrk** [comment]
How's it look? The branch is [issue103](https://github.com/NICMx/FORT-validator/tree/issue103).

**Comment by rfc1036** [comment]
Looks good to me, thank you!

Still log-related: I see in the log a lot of repeated messages like this one:

```
Dec 04 23:44:54 bongo.bofh.it fort[581896]: /etc/tals/apnic.tal: Cannot generate rsync://rpki.sub.apnic.net/repository/'s cage. I'm probably going to end up deleting it from the cache.
```

It is not totally clear to me if this means that the source repository is broken (but then is it worth frequently logging this every at a non-debug priority?) or that the local system is broken.

**Comment by ydahhrk** [comment]
It's a bug; I'll investigate.

**Comment by ydahhrk** [comment]
It happens during cache cleanup. Fort is trying to infer cages (jails/containers) out of a list of URLs; Some of them are rsync, and not supposed to be caged.

So it's a false alarm. It doesn't delete anything, because those cages didn't exist in the first place.

I'll fix it.

**Comment by ydahhrk** [comment]
Ready; ping.

**Comment by ydahhrk** [comment]
Once I've merged #103, I can see myself releasing a new version on December 15, assuming no new issues hold me back for too long.

**Comment by ydahhrk** [comment]
1.6.1 released; closing.
